pkg/kudoctl/kube/config.go

```diff
@@ -4,21 +4,27 @@ import (
 	"fmt"
 
 	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	"github.com/kudobuilder/kudo/pkg/apis"
+	"github.com/kudobuilder/kudo/pkg/client/clientset/versioned"
 	"github.com/kudobuilder/kudo/pkg/kudoctl/clog"
 )
 
 // Client provides access different K8S clients
 type Client struct {
+	Scheme        *runtime.Scheme
```

Comment:
Is there a reason to expose the scheme here? I couldn't find any code using this field.

Reply:
Yeah, I think I used it at some point and then cleaned it up so it's not used anymore. I've removed the field